CHANGELOG.md

```diff
@@ -6,6 +6,24 @@ All user visible changes to this project will be documented in this file. This p
 
 
 
+## [0.6.2] · 2024-??-?? (unreleased)
+[0.6.2]: /../../tree/medea-jason-0.6.2
+
+[Diff](https://github.com/instrumentisto/medea-jason/compare/medea-jason-0.6.1...medea-jason-0.6.2)
+
+### Changed
+
+- Replaced [wee_alloc] WASM allocator with [talc]. ([#187])
+
+### BC Breaks
+
+- Replaced `wee_alloc` feature with a `talc`. ([#187])
```

Comment:
Technically, this is a breaking change, so in a perfect world we should do a minor version bump here, but because this BC is too minor, I propose just doing a patch version bump.

Reply:
@evdokimovs how this is a breaking change?

Reply:
@evdokimovs nothing wrong with bumping minor here.

Reply:
@tyranron ,

It's a feature, so if someone is using our crate with a `wee_alloc` feature in a features list, then after this update, their `Cargo.toml` will be broken. Because it can break something - it's BC as I see it.

Reply:
@tyranron ,

Here is example:

_`foobar/Cargo.toml`_
```toml
[package]
name = "foobar"
version = "0.1.0"
edition = "2021"

[dependencies]
talc = { version = "4.4.1", optional = true }
```

_`bar/Cargo.toml`_
```toml
[package]
name = "bar"
version = "0.1.0"
edition = "2021"

[dependencies]
foobar = { path = "../foobar", features = ["wee_alloc"] }
```

Build output:

```
error: failed to select a version for `foobar`.
    ... required by package `bar v0.1.0 (/private/tmp/bar)`
versions that meet the requirements `*` (locked to 0.1.0) are: 0.1.0

the package `bar` depends on `foobar`, with features: `wee_alloc` but `foobar` does not have these features.


failed to select a version for `foobar` which could resolve this conflict
```